Hudl.FFmpeg/Settings/FrameRateInput.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using Hudl.FFmpeg.Attributes;
+using Hudl.FFmpeg.Enums;
+using Hudl.FFmpeg.Resources.BaseTypes;
+using Hudl.FFmpeg.Settings.Attributes;
+using Hudl.FFmpeg.Settings.BaseTypes;
+
+namespace Hudl.FFmpeg.Settings
+{
+    /// <summary>
+    /// Set frame rate (Hz value, fraction or abbreviation).
+    /// </summary>
+    [ForStream(Type = typeof(VideoStream))]
+    [Setting(Name = "r", ResourceType = SettingsCollectionResourceType.Input)]
+    public class FrameRateInput : BaseFrameRate
+    {
+        public FrameRateInput()
+        {
+        }
+        public FrameRateInput(double rate)
+        {
+            if (rate <= 0)
```

Comment:
Seems like you don't even need this code here and can instead just call the base constructor.